src/main/resources/com/google/api/codegen/py/method_sample.snip

```diff
@@ -164,7 +94,7 @@
 @end
 
 @snippet optionalArrayMethodSampleCodeServerStreaming(apiMethod, initCode)
```

Comment:
These `@snippet`s should go back to being `@private` as they were before we used them for the standalone stubs (cf #2046 ). This will help guarantee we don't accidentally call the wrong one. Pretty much each of the functions you coped to the standalone file should be made `@private` here too.

Reply:
Done